web/src/utils/storyblok.ts

```diff
@@ -54,6 +54,30 @@ export const defaultPodcastImage =
 export const GENERIC_ERROR_MESSAGE =
 	"Oops! Something went wrong and we're working to fix it. Please try again later.";
 
+//Fetch cache version from storyblok space API
+export const fetchCacheVersion = async (): Promise<number> => {
```

Comment:
Just double checking that this is doing what we expected with regards to caching (i.e. CV value is being cached and we're not hitting SB API for every request as we were worried about initially)?

Reply:
I _think_ we were expecting it to cache/retrieve from Redis when its going via next-web-api like the other API requests for stories / links / etc do? In hindsight should've captured this as an AC!

Reply:
Should be able to check if this is happening or if we're hitting SB every time with the additional logging Colin has added to next-web-api